connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1135,11 +1146,12 @@ ble_error_t Gap::reset()
         _connectable_payload_size_exceeded.clear();
         _set_is_connectable.clear();
 
+#if BLE_FEATURE_EXTENDED_ADVERTISING
```

Comment:
this is not human parsable anymore, I can't even work out if the parens below creates a mismatch

Reply:
I will rework this one. I agree it is a mess.

Reply:
done.